cmd/jaeger/internal/integration/storagecleaner/README.md

```diff
@@ -0,0 +1,47 @@
+# storage_cleaner
+
+This module implements an extension that allows purging the backend storage by exposing a HTTP endpoint for making POST requests. 
+
+The storage_cleaner extension is intended to be used only in tests, providing a way to clear the storage between test runs. Making a POST request to the exposed endpoint will delete all data in storage.
+
+
+```mermaid
+flowchart LR
+    Receiver --> Processor
+    Processor --> Exporter
+    JaegerStorageExension -->|"(1) get storage"| Exporter
+    Exporter -->|"(2) write trace"| Badger
```

Comment:
suggest replacing `Badger` with `Storage`, and `Badger_e2e_test` with `E2E_test`